ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -270,61 +269,99 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
     __experimentalEChartsOptionsOverride,
     noDataVariant,
     timeZone,
-    tooltipPinnedCoords,
     isStackedBar,
+    enablePinning,
+    pinnedCrosshair,
   ]);
 
+  // Update adjacent charts so tooltip is unpinned when current chart is clicked.
+  useEffect(() => {
+    // Only allow pinning one tooltip at a time, subsequent tooltip click unpins previous.
+    // Multiple tooltips can only be pinned if Ctrl or Cmd key is pressed while clicking.
+    if (tooltipPinnedCoords !== null && lastTooltipPinnedCoords !== null) {
```

Comment:
```suggestion
    const multipleTooltipsPinned = tooltipPinnedCoords !== null && lastTooltipPinnedCoords !== null && !isEqual(lastTooltipPinnedCoords, tooltipPinnedCoords);
    if (multipleTooltipsPinned) 
```

nit: not a big deal but i find the code easier to read if we create a variable for the conditional statements 😄

Reply:
Nice, yeah I like this